x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -249,12 +248,8 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
   );
   const signalIndexNameSourcerer = useDeepEqualSelector(getSignalIndexNameSelector);
   const defaultDataView = useDeepEqualSelector(getDefaultDataViewSelector);
-  const { isSignalIndexNeedsInit, showResults } = useMemo(
-    () => ({
-      isSignalIndexNeedsInit:
-        indicesExist === false && !defaultDataView.title.includes(`${signalIndexNameSourcerer}`),
-      showResults: indicesExist || defaultDataView.title.includes(`${signalIndexNameSourcerer}`),
-    }),
+  const isSignalIndexNeedsInit = useMemo(
+    () => !indicesExist && !defaultDataView.title.includes(`${signalIndexNameSourcerer}`),
```

Comment:
signalIndexNameSourcerer seems could be null, although I guess defaultDataView.title could never includes `null` string, it just a bit weird to covert null to string.

Reply:
yes its for a type reason, cant check if a string includes null

Reply:
will do